src/connector/src/source/iceberg/mod.rs

```diff
@@ -266,7 +344,6 @@ impl IcebergSplitEnumerator {
                     data_files.push(IcebergFileScanTaskJsonStr::serialize(&task));
                 }
                 iceberg::spec::DataContentType::EqualityDeletes => {
-                    task.project_field_ids = task.equality_ids.clone();
```

Comment:
Why remove these field ids?

Reply:
Because after this pr, we can make sure that the filed id from the select() step is the final id, so doesn't need to be changed here